boards/CMakeLists.txt

```diff
@@ -284,7 +284,12 @@ function(compile_googletest_executable
         ARM_BINARY_X86_COMPATIBLE_SRCS
         ARM_BINARY_INCLUDE_DIRS
         )
-    set(TEST_EXECUTABLE_NAME "${BOARD_NAME}_test")
+    if(CMAKE_HOST_SYSTEM_NAME STREQUAL "Windows")
+        set(TEST_EXECUTABLE_NAME "${BOARD_NAME}_test.exe")
```

Comment:
This is needed because windows generates an .exe at the end of the board test executables